src/client/components/app/LabDetail/LabDetailPanels.tsx

```diff
@@ -0,0 +1,287 @@
+  currentNetwork,
+  blockExplorerUrl,
+}: LabDetailPanelsProps) => {
+  const { t } = useAppTranslation('labdetails');
+
+  const context = useContext(AppContext);
+
+  const handleAddToken = () => {
+    const {
+      address,
+      decimals,
+      displayIcon,
+      token: { symbol },
+    } = selectedLab;
+
+    if (context?.wallet.addToken) {
+      context?.wallet.addToken(address, symbol.substring(0, 11), Number(decimals), displayIcon || '');
+    }
+  };
+
+  return (
+    <>
+      <Row>
+        <LabOverview>
+          <StyledCardHeaderContainer>
+            <StyledCardHeader header={t('labdetails:overview-panel.header')} />
+            {displayAddToken ? (
+              <RelativeContainer onClick={handleAddToken}>
+                <StyledImg src={MetamaskLogo} />
+                <IconOverImage Component={AddCircleIcon} />
+              </RelativeContainer>
+            ) : null}
+            <ScanNetworkIcon
+              currentNetwork={currentNetwork}
+              blockExplorerUrl={blockExplorerUrl}
+              address={selectedLab.address}
+            />
+          </StyledCardHeaderContainer>
+
+          <OverviewTokenInfo>
+            <TokenLogo variant="background">
+              <TokenIcon icon={selectedLab.displayIcon} symbol={selectedLab.displayName} size="xBig" />
+            </TokenLogo>
+
+            <TokenInfo>
+              <InfoValueTitle>{selectedLab?.displayName}</InfoValueTitle>
+
+              <InfoValueRow>
+                <span>{t('labdetails:overview-panel.apy')}</span>
+                <TextWithIcon>
+                  <StyledText fontWeight="bold">
+                    <span>{formatApy(selectedLab.apyData)}</span>
+                  </StyledText>
+                  {getTooltip({
+                    apyMetadata: selectedLab.apyMetadata,
+                    address: selectedLab.address,
+                  })}
+                </TextWithIcon>
+              </InfoValueRow>
+              <InfoValueRow>
+                <span>{t('labdetails:overview-panel.total-assets')}</span>
+                <StyledText>{humanize('usd', selectedLab.labBalanceUsdc, USDC_DECIMALS, 0)}</StyledText>
+              </InfoValueRow>
+              <InfoValueRow>
+                <span>{t('labdetails:overview-panel.type')}</span>
+                <StyledText>{selectedLab.token.categories}</StyledText>
+              </InfoValueRow>
+              <InfoValueRow>
+                <span>{t('labdetails:overview-panel.web')}</span>
+                <StyledLink href={selectedLab.token.website}>{selectedLab.token.website}</StyledLink>
+              </InfoValueRow>
+            </TokenInfo>
+          </OverviewTokenInfo>
+
+          <StyledCardHeader header={t('labdetails:overview-panel.about')} />
+
+          {selectedLab.token.description && (
+            <OverviewInfo variant="surface" cardSize="micro">
+              <StyledCardContent>
+                <Markdown>{selectedLab.token.description}</Markdown>
```

Comment:
Beware places that `selectedLab.token` is used. In Vaults page the displayed data is mostly of the underlying token. In Labs we do have to display the vault data itself.

Reply:
Does this mean we might need to get the `description` from somewhere else? `selectedLab.underlyingToken` or similar?

Reply:
`token` already holds the information of the underlying token. Not sure what is the best route here. Maybe `LabView` should have additional properties as needed.

Reply:
Shall we leave that to another PR?

Reply:
Lets try to keep it here since releasing it without all the info, makes the use case of having this view very limited.